File: 3107. Minimum Operations to Make Median of Array Equal to K.go
func minOperationsToMakeMedianK(nums []int, k int) int64 {
	sort.Ints(nums)
	mid := len(nums) / 2
	var answer int64
	if nums[mid] < k {
		for i := mid; i < len(nums); i++ {
			if nums[i] >= k {
				break
			}
			answer += int64(k - nums[i])
		}
	} else if nums[mid] > k {
		for i := mid; i >= 0; i-- {
			if nums[i] <= k {
				break
			}
			answer += int64(nums[i] - k)
		}
	}
	return answer
}

File: 3107. Minimum Operations to Make Median of Array Equal to K.py
class Solution:
    def minOperationsToMakeMedianK(self, nums: List[int], k: int) -> int:
        nums.sort()
        mid = len(nums) // 2
        answer = 0
        if nums[mid] < k:
            for i in range(mid, len(nums)):
                if (nums[i] >= k):
                    break
                answer += k - nums[i]
        elif nums[mid] > k:
            for i in range(mid, -1, -1):
                if nums[i] <= k:
                    break
                answer += nums[i] - k
        return answer